ghcide/src/Development/IDE/Types/Exports.hs

```diff
@@ -178,7 +178,7 @@ unpackAvail mn
   | otherwise = const []
   where
     !mod = pack $ moduleNameString mn
-    f id@IdentInfo {..} = (pack (prettyPrint name), moduleNameText,[id])
+    f id@IdentInfo {..} = (pack (occNameString name), moduleNameText,[id])
```

Comment:
why different here?

Reply:
The same one, just a lonely clear name.

Reply:
I have no idea if this is sensible or not, I don't know this bit of the code! Just wondering. Probably could do with a comment.

Reply:
Replaced with `printOutputable`